### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mc-usds**
Creating a pull request template to help teams know what information they should include when submitting a code review.

### CODE REVIEW

**Comment by mc-usds** [comment]
a pull request already exists. will update that instead an create a new pull request.

**Comment by mc-usds** [comment]
Reopening. Did not mean to close the vets-website PR, I meant to close the vets-api PR.

**Comment by mc-usds** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group 
Need someone to review and approve

**Comment by jkassemi** [comment]
Just curious, aside from screenshots, what accounts for the differences between this and https://github.com/department-of-veterans-affairs/vets-api/pull/2199/?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Description
+Please explain the changes you made here.
+
+### Testing done
```
Comment (cehsu) [inline PULL_REQUEST_TEMPLATE.md]: Should we include updating documentation to this list?

There are also a number of comments on API template that may also make sense to include here:
```
- is there a feature flag for this change, and if so, does the devops team need to be made aware of varying configuration for each environment?
...
- are there dashboards and associated instrumentation in place for this change? Have alerts been configured?
- Which alerts would indicate an issue with this functionality?
```

Similar to the alert comment above, error logging in Sentry might be helpful to include here.


We also don't have an automated solution in place for cross-browser testing, perhaps a section for `#### How should this be manually tested?` could help address that kind of gap.

**Review by cehsu — APPROVED** [review]
LGTM. I had a few questions/comments for your consideration.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mc-usds) [reply PULL_REQUEST_TEMPLATE.md]: @jkassemi  That was a oversight on my part. When I created this one, I didn't know there was an existing pull request template in the vets-api repository. I've updated this one to match the one in vets-api.

@cehsu I've updated the Testing done and Testing Planned html comments to mention manual testing and cross browser testing. And also included the Acceptance Criteria section that is also in the vets-api pull request template.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Acceptance Criteria (Definition of Done)
+
+#### Unique to this PR
+<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
+- [ ] _Replace this line with individual tasks unique to your PR_
+
+#### Applies to all PRs
+
+- [ ] Appropriate logging
+- [ ] Swagger docs have been updated, if applicable
```
Comment (cehsu) [inline PULL_REQUEST_TEMPLATE.md]: Vets-website doesn't use swagger for documentation, and documentation is housed in several locations (e.g. `https://github.com/usds/us-forms-system/tree/master/src/js` and the `/docs` folder on this repo), should we list those (perhaps as a checklist) instead here or simply say `update documentation`?

Comment (mc-usds) [reply PULL_REQUEST_TEMPLATE.md]: Thanks for pointing this out. I've updated the checklist item to be generic - documentation has been updated.

**Pull request merged**